libs/base/ui/overlays/dropdown/dropdown.component.spec.ts

```diff
@@ -52,6 +52,48 @@ describe('DropdownComponent', () => {
     });
   });
 
+  describe('when openOnHover property is assigned', () => {
+    beforeEach(async () => {
+      element = await fixture(
+        html`<oryx-dropdown openOnHover></oryx-dropdown>`
```

Comment:
maybe just **hover**?
```suggestion
        html`<oryx-dropdown hover></oryx-dropdown>`
```

Reply:
hover sounds like an event, rather than a configuration to react to such event.